commands/roles.go

```diff
@@ -72,6 +72,8 @@ func rolesSystemAdminCmdF(c client.Client, _ *cobra.Command, args []string) erro
 
 			printer.Print(fmt.Sprintf("System admin role assigned to user %q", args[i]))
 		}
+
+		printer.Print(fmt.Sprintf("Updated %s roles: %s", user.Email, user.Roles))
```

Comment:
The roles that `user` contains are not updated with the new information that resulted from the `c.UpdateUserRoles`, so this will only print whatever roles the user had before making that request.

Instead, we could use the roles in the `roles` list that we have built before, assuming that if the request succeeded, the user will have them applied.

One last point would be that we should print this information as part of the `printer.Print` call in line 73, as this will only get printed if the request succeeds. So maybe something like:

```suggestion
			printer.Print(fmt.Sprintf("System admin role assigned to user %q. Current roles are: %s", args[i], strings.Join(", ", roles))
		}
```